package.json

```diff
@@ -51,7 +48,10 @@
     "@neume-network/message-schema": "0.3.1"
   },
   "peerDependencies": {
-    "eth-fun": "0.9.x"
+    "eth-fun": "0.9.x",
+    "@neume-network/schema": "0.3.x",
+    "ajv": "8.x.x",
+    "ajv-formats": "2.x.x"
```

Comment:
Since, these packages were required in neume-network/core I have added them to devDependency in neume-network/core and made them peerDependencies here.